src/components/Bridge/BridgeCard/index.tsx

```diff
@@ -181,6 +181,7 @@ const BridgeCard: React.FC<BridgeCardProps> = (props) => {
           method: 'wallet_switchEthereumChain',
           params: [{ chainId: `0x${chain?.chain_id?.toString(16)}` }],
         });
+        window.location.reload();
```

Comment:
Yup, is there a way around it?

Reply:
Yes. Normally, It's okay if the user jumps between non-supported chains (avalanche and songbird). (We don't need to reload it) But there are so many hooks related to the supported chain so I chose this fastest way instead of handling it. We can publish it with this fastest solution for now, and I'll take care of this after 0xSquid integration. Is it okay?

Reply:
sure @bilalcorbacioglu